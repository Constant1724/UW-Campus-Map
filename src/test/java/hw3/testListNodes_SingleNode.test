# START_TEST
# NAME SingleNode
# test if we can listNode in the graph.
# Create a graph
CreateGraph graph1

# Add single node
AddNode graph1 Content1

# Print the nodes in the graph
ListNodes graph1
# END_TEST