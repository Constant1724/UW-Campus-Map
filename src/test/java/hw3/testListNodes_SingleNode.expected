# START_TEST
# NAME SingleNode
# test if we can listNode in the graph.
# Create a graph
created graph graph1

# Add single node
added node Content1 to graph1

# Print the nodes in the graph
graph1 contains: Content1
# END_TEST